### ISSUE TITLE
Layout/LineContinuationLeadingSpace unsupported

### ISSUE THREAD

**Issue by StevenHarlow** [misogi/vscode-ruby-rubocop#169]
I recently installed the plugin and got this warning:

```
Error: unrecognized cop or department Layout/LineContinuationLeadingSpace found in .rubocop.yml Did you mean `Layout/MultilineOperationIndentation`? unrecognized cop or department Layout/LineContinuationSpacing found in .rubocop.yml Did you mean `Layout/MultilineOperationIndentation`?
```

However, [Layout/LineContinuationLeadingSpace](https://docs.rubocop.org/rubocop/cops_layout.html#layoutlinecontinuationleadingspace) is a valid cop
